Source/WebCore/rendering/style/BasicShapes.h

```diff
@@ -179,7 +189,7 @@ class BasicShapeCircle final : public BasicShape {
     const BasicShapeCenterCoordinate& centerX() const { return m_centerX; }
     const BasicShapeCenterCoordinate& centerY() const { return m_centerY; }
     const BasicShapeRadius& radius() const { return m_radius; }
-    float floatValueForRadiusInBox(float boxWidth, float boxHeight) const;
+    float floatValueForRadiusInBox(float boxWidth, float boxHeight, float centerX, float centerY) const;
```

Comment:
Any time you pass a width, height pair ask yourself if passing a Size type would make more sense.

Reply:
Fixed.